bundles/org.openhab.binding.enturno/README.md

```diff
@@ -0,0 +1,141 @@
+# EnturNo binding
+
+{% include base.html %}
+
+This binding is meant to get Norwegian public transport real-time (estimated) data from [Entur.no API](https://developer.entur.org/content/journey-planner-0).
+
+It runs GraphQL-API query against the service.
```

Comment:
Please think of the readme file as being the user documentation to the binding, not the developer documentation.

Can you please remove implementation specific parts and move them to a CONTRIBUTE.md file?

The readme should be like an advertiser to your binding. Describe what it is capable of (you can quote the official website) and include pictures if you like to.

Reply:
Done